packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -7,6 +7,7 @@ import {
   WebhookTriggerEvents,
 } from "@prisma/client";
 import async from "async";
+import { isValidPhoneNumber } from "libphonenumber-js";
```

Comment:
Let's add `"libphonenumber-js": "^1.10.12",` to `packages/features/package.json`